pkg/provider/azure_vmssflex.go

```diff
@@ -156,17 +150,51 @@ func (fs *FlexScaleSet) GetVMSetNames(service *v1.Service, nodes []*v1.Node) (*[
 	return vmssFlexNames, nil
 }
 
+// GetNodeNameByProviderID gets the node name by provider ID.
+func (fs *FlexScaleSet) GetNodeNameByProviderID(providerID string) (types.NodeName, error) {
```

Comment:
it's better providing a `providerID` example in the func comments.

Reply:
done